app/models/restaurant.rb

```diff
@@ -0,0 +1,2 @@
+class Restaurant < ApplicationRecord
+end
```

Comment:
Please add some input validation. Name of restaurant should probably be unique, most of the other inputs also have limitations.